### ISSUE TITLE
Checkbox widget does not render description

### ISSUE THREAD

**Issue by sverhoeven** [i-VRESSE/workflow-builder#61]
![localhost_3000_ (4)](https://user-images.githubusercontent.com/1713488/161565755-fe027bb8-be5d-4e83-b023-74d617ab6264.png)
while in catalog the parameter looks like

```yaml
      mol_fix_origin:
        type: array
        items:
          default: false
          title: Fix molecule
          description: Fixes the molecule in its original position.
          $comment: Fixes the molecule in its original position.
          type: boolean
```

I expected the ` Fixes the molecule in its original position.` to be shown near the checkbox..

**Comment by Peter9192** [comment]
Could this be a nice first issue for @bvreede (and yours truly)?

**Comment by sverhoeven** [comment]
Could be, it would be overwriting the default CheckBoxWidget with one of our one. See https://react-jsonschema-form.readthedocs.io/en/latest/advanced-customization/custom-widgets-fields/

The widget in the default them at https://github.com/rjsf-team/react-jsonschema-form/blob/master/packages/core/src/components/widgets/CheckboxWidget.js#L27-L29 does print description 

While the bootstrap4 widget at https://github.com/rjsf-team/react-jsonschema-form/blob/master/packages/bootstrap-4/src/CheckboxWidget/CheckboxWidget.tsx does not.

It would be nice to fix it upstream in https://github.com/rjsf-team/react-jsonschema-form, but the maintainers have quite a few PRs they still have to merge so it will most likely take quite a while to get our own PR merged.

**Comment by bvreede** [comment]
Fixed with #64 🎉
